internal/cmd/beta/logs/instance/list/list.go

```diff
@@ -0,0 +1,153 @@
+	Limit *int64
+}
+
+const (
+	limitFlag = "limit"
+)
+
+func NewCmd(params *types.CmdParams) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "Lists Logs instances",
+		Long:  "Lists Logs instances within the project.",
+		Args:  args.NoArgs,
+		Example: examples.Build(
+			examples.NewExample(
+				`List all Logs instances`,
+				`$ stackit beta logs instance list`,
+			),
+			examples.NewExample(
+				`List the first 10 Logs instances`,
+				`$ stackit beta logs instance list --limit=10`,
+			),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+			model, err := parseInput(params.Printer, cmd, args)
+			if err != nil {
+				return err
+			}
+
+			// Configure API client
+			apiClient, err := client.ConfigureClient(params.Printer, params.CliVersion)
+			if err != nil {
+				return err
+			}
+
+			projectLabel, err := projectname.GetProjectName(ctx, params.Printer, params.CliVersion, cmd)
+			if err != nil {
+				params.Printer.Debug(print.ErrorLevel, "get project name: %v", err)
+				projectLabel = model.ProjectId
+			} else if projectLabel == "" {
+				projectLabel = model.ProjectId
+			}
+
+			// Call API
+			request := buildRequest(ctx, model, apiClient)
+
+			response, err := request.Execute()
+			if err != nil {
+				return fmt.Errorf("list Logs instances: %w", err)
+			}
+			items := response.GetInstances()
+
+			// Truncate output
+			if model.Limit != nil && len(items) > int(*model.Limit) {
+				items = items[:*model.Limit]
+			}
+
+			return outputResult(params.Printer, model.OutputFormat, projectLabel, items)
+		},
+	}
+
+	configureFlags(cmd)
+	return cmd
+}
+
+func configureFlags(cmd *cobra.Command) {
+	cmd.Flags().Int64(limitFlag, 0, "Limit the output to the first n elements")
+}
+
+func parseInput(p *print.Printer, cmd *cobra.Command, _ []string) (*inputModel, error) {
+	globalFlags := globalflags.Parse(p, cmd)
+	if globalFlags.ProjectId == "" {
+		return nil, &errors.ProjectIdError{}
+	}
+
+	limit := flags.FlagToInt64Pointer(p, cmd, limitFlag)
+	if limit != nil && *limit < 1 {
+		return nil, &errors.FlagValidationError{
+			Flag:    limitFlag,
+			Details: "must be greater than 0",
+		}
+	}
+
+	model := inputModel{
+		GlobalFlagModel: globalFlags,
+		Limit:           limit,
+	}
+
+	p.DebugInputModel(model)
+	return &model, nil
+}
+
+func buildRequest(ctx context.Context, model *inputModel, apiClient *logs.APIClient) logs.ApiListLogsInstancesRequest {
+	request := apiClient.ListLogsInstances(ctx, model.ProjectId, model.Region)
+
+	return request
+}
+func outputResult(p *print.Printer, outputFormat, projectLabel string, instances []logs.LogsInstance) error {
+	return p.OutputResult(outputFormat, instances, func() error {
+		if len(instances) == 0 {
+			p.Outputf("No Logs instances found for project %q", projectLabel)
+			return nil
+		}
+
+		table := tables.NewTable()
+		table.SetHeader("NAME", "ID", "STATUS")
+		for i := range instances {
```

Comment:
```suggestion
		for _, instance := range instances {
```

Saves yourself the next line (`instance := instances[i]`)...